versionedRouter.js

```diff
@@ -106,16 +110,21 @@ async function handleDest(ctx, version, destination) {
   });
   const respList = [];
   const executeStartTime = new Date();
+  let destHandler;
+  // destination definition is going to be same for any kind of event
```

Comment:
Some inline documentation for this would make things easier to understand

Reply:
Can you now check if this is making it easier to understand ?